radlab-ui/webapp/src/routes/Deployments.tsx

```diff
@@ -36,6 +40,39 @@ const Deployments: React.FC<DeploymentsProps> = () => {
   const [deploymentTab, setDeploymentTab] = useState(
     isAdmin ? DEPLOYMENT_TAB.ALL : DEPLOYMENT_TAB.MINE,
   )
+  const filteredDeployments = deploymentStore(
+    (state) => state.filteredDeployments,
+  )
+  const setDeployments = deploymentStore((state) => state.setDeployments)
+  const [modules, setModules] = useState<IModule[] | null>(null)
+
+  const deletedAt = "DELETED"
+
+  const activeStatuses = [
+    DEPLOYMENT_STATUS.SUCCESS,
+    DEPLOYMENT_STATUS.PENDING,
+    DEPLOYMENT_STATUS.QUEUED,
+    DEPLOYMENT_STATUS.FAILURE,
+    DEPLOYMENT_STATUS.EXPIRE,
+    DEPLOYMENT_STATUS.CANCELLED,
+    DEPLOYMENT_STATUS.INTERNAL_ERROR,
+    DEPLOYMENT_STATUS.STATUS_UNKNOWN,
+    DEPLOYMENT_STATUS.TIMEOUT,
+    DEPLOYMENT_STATUS.WORKING,
+    deletedAt,
+  ]
```

Comment:
Can you simplify to something like:
```ts
  const activeStatuses = [
    ...DEPLOYMENT_STATUS,
    deletedAt,
  ]
```

Reply:
DEPLOYMENT_STATUS is not an array actually so we can not use spread operator `...DEPLOYMENT_STATUS`. Any other way?

Reply:
Can you define it first with `zod` and then generate the type with via `z.infer` ([docs](https://github.com/colinhacks/zod#zod-enums))

Reply:
Looks good. Thanks!